modules/nw-route-specific-annotations.adoc

```diff
@@ -70,6 +70,27 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 Environment variables cannot be edited.
 ====
 
+.Router timeout variables
+
+TimeUnits are represented by a number followed by the unit: us *(microseconds), ms (milliseconds, default), s (seconds), m (minutes), h *(hours), d (days).
```

Comment:
TimeUnits either needs to be enclosed in backticks or separated into two words, depending on if it's a system parameter or just a description.

Reply:
Acked